foundations/02-class-id-selectors/solution/solution.css

```diff
@@ -14,6 +14,5 @@
 
 #four {
```

Comment:
Since this is still relatively early and users may not have had a chance to gain an understanding of reducing code duplication, I think it would be worth adding a comment above the `#four` block along the lines of "Instead of applying the `adjust-font-size` class to this element in the HTML file, you could have also simply added a rule to the `#four` selector to set the font size. Reusing the class on this element helps reduce duplicate code."

Reply:
I agree as well because it isn't apparent that the class `adjust-font-size` is meant to reduce code duplication and new users will miss it out.